packages/manager/src/features/linodes/LinodesCreate/TabbedContent/FromAppsContent.tsx

```diff
@@ -221,6 +226,7 @@ class FromAppsContent extends React.Component<CombinedProps, State> {
   };
 
   handleSelectCategory = (categoryItem: Item<AppCategory>) => {
+    sendMarketplaceSearchEvent('Category Dropdown', categoryItem.label);
```

Comment:
Clearing the category gives an error. This optional chain should fix it! Maybe we shouldn't send an event if the input is cleared? 

https://user-images.githubusercontent.com/6440455/187231818-5ec7f11a-a970-4d6b-9fc1-e0a0dec3ecd3.mov


```suggestion
    sendMarketplaceSearchEvent('Category Dropdown', categoryItem?.label);
```